<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1260</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::TabShape::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1260</width>
      <height>680</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_home">
     <attribute name="title">
      <string>Home</string>
     </attribute>
     <widget class="QGroupBox" name="gb_serial">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>221</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="lbchannel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>5</number>
       </property>
       <property name="text">
        <string>Channel</string>
       </property>
       <property name="textFormat">
        <enum>Qt::TextFormat::AutoText</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="serConnect">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>91</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Serial Connect</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbport">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>5</number>
       </property>
       <property name="text">
        <string>Port</string>
       </property>
       <property name="textFormat">
        <enum>Qt::TextFormat::AutoText</enum>
       </property>
      </widget>
      <widget class="QComboBox" name="cbPort">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cbChannel">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>4800</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>115200</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="serDisConnect">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>80</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="lblBgImage">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>0</y>
        <width>1011</width>
        <height>641</height>
       </rect>
      </property>
      <property name="text">
       <string>Image</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_data">
     <attribute name="title">
      <string>Data</string>
     </attribute>
     <widget class="QGroupBox" name="gb_IMU_info">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>0</y>
        <width>311</width>
        <height>621</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QProgressBar" name="imu_load_status">
       <property name="geometry">
        <rect>
         <x>147</x>
         <y>10</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
      <widget class="QLabel" name="lbstatus">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>5</number>
       </property>
       <property name="text">
        <string>Status connect IMU</string>
       </property>
       <property name="textFormat">
        <enum>Qt::TextFormat::AutoText</enum>
       </property>
      </widget>
      <widget class="QLabel" name="lbgyro">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>5</number>
       </property>
       <property name="text">
        <string>Address GYRO</string>
       </property>
       <property name="textFormat">
        <enum>Qt::TextFormat::AutoText</enum>
       </property>
      </widget>
      <widget class="QLabel" name="dataconfig">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>5</number>
       </property>
       <property name="text">
        <string>Data configure IMU</string>
       </property>
       <property name="textFormat">
        <enum>Qt::TextFormat::AutoText</enum>
       </property>
      </widget>
      <widget class="QLabel" name="lbslave">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>5</number>
       </property>
       <property name="text">
        <string>Address Slave</string>
       </property>
       <property name="textFormat">
        <enum>Qt::TextFormat::AutoText</enum>
       </property>
      </widget>
      <widget class="QLabel" name="lbyaw">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>5</number>
       </property>
       <property name="text">
        <string>Yaw (Z Rot)</string>
       </property>
       <property name="textFormat">
        <enum>Qt::TextFormat::AutoText</enum>
       </property>
      </widget>
      <widget class="QLabel" name="RotationInfo">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>171</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>5</number>
       </property>
       <property name="text">
        <string>Data rotation information</string>
       </property>
       <property name="textFormat">
        <enum>Qt::TextFormat::AutoText</enum>
       </property>
      </widget>
      <widget class="QLabel" name="lbroll">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>5</number>
       </property>
       <property name="text">
        <string>Roll (X Rot)</string>
       </property>
       <property name="textFormat">
        <enum>Qt::TextFormat::AutoText</enum>
       </property>
      </widget>
      <widget class="QLabel" name="lbpitch">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>5</number>
       </property>
       <property name="text">
        <string>Pitch (Y Rot)</string>
       </property>
       <property name="textFormat">
        <enum>Qt::TextFormat::AutoText</enum>
       </property>
      </widget>
      <widget class="QLabel" name="RotationInfo_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>310</y>
         <width>171</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>5</number>
       </property>
       <property name="text">
        <string>Euler angle information</string>
       </property>
       <property name="textFormat">
        <enum>Qt::TextFormat::AutoText</enum>
       </property>
      </widget>
      <widget class="QLabel" name="lbrollEuler">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>340</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>5</number>
       </property>
       <property name="text">
        <string>RollE (X Rot)</string>
       </property>
       <property name="textFormat">
        <enum>Qt::TextFormat::AutoText</enum>
       </property>
      </widget>
      <widget class="QLabel" name="lbpitchEuler">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>370</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>5</number>
       </property>
       <property name="text">
        <string>PitchE (Y Rot)</string>
       </property>
       <property name="textFormat">
        <enum>Qt::TextFormat::AutoText</enum>
       </property>
      </widget>
      <widget class="QLabel" name="lbyawEuler">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>400</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>5</number>
       </property>
       <property name="text">
        <string>YawE (Z Rot)</string>
       </property>
       <property name="textFormat">
        <enum>Qt::TextFormat::AutoText</enum>
       </property>
      </widget>
      <widget class="QLabel" name="lbpitchqua">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>510</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>5</number>
       </property>
       <property name="text">
        <string>PitchQ (Y Rot)</string>
       </property>
       <property name="textFormat">
        <enum>Qt::TextFormat::AutoText</enum>
       </property>
      </widget>
      <widget class="QLabel" name="lbyawqua">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>540</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>5</number>
       </property>
       <property name="text">
        <string>YawQ (Z Rot)</string>
       </property>
       <property name="textFormat">
        <enum>Qt::TextFormat::AutoText</enum>
       </property>
      </widget>
      <widget class="QLabel" name="lbrollqua">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>480</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>5</number>
       </property>
       <property name="text">
        <string>RollQ (X Rot)</string>
       </property>
       <property name="textFormat">
        <enum>Qt::TextFormat::AutoText</enum>
       </property>
      </widget>
      <widget class="QLabel" name="RotationInfo_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>450</y>
         <width>171</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>5</number>
       </property>
       <property name="text">
        <string>Quaternion angle information</string>
       </property>
       <property name="textFormat">
        <enum>Qt::TextFormat::AutoText</enum>
       </property>
      </widget>
      <widget class="QLabel" name="lbacc">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>5</number>
       </property>
       <property name="text">
        <string>Address ACC</string>
       </property>
       <property name="textFormat">
        <enum>Qt::TextFormat::AutoText</enum>
       </property>
      </widget>
      <widget class="QSpinBox" name="lblGyroInfo">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>70</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="lblAccInfo">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>100</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="lblSlaveAddr">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>130</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="lblRoll">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>210</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="lblPitch">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>239</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="lblYaw">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>270</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="lblPitchEuler">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>371</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="lblYawEuler">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>400</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="lblRollEuler">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>342</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="lblYawQua">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>540</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="lblRollQua">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>482</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="lblPitchQua">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>511</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>0</y>
        <width>211</width>
        <height>621</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="lbstatus_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>5</number>
       </property>
       <property name="text">
        <string>Serial Data Input</string>
       </property>
       <property name="textFormat">
        <enum>Qt::TextFormat::AutoText</enum>
       </property>
      </widget>
      <widget class="QTextEdit" name="serialtable">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>191</width>
         <height>571</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1260</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
